ios/FluentUI.Demo/FluentUIDemoTests/TooltipTest.swift

```diff
@@ -12,4 +12,29 @@ class TooltipTest: BaseTest {
     func testLaunch() throws {
         XCTAssertTrue(app.navigationBars[controlName].exists)
     }
+
+    func testMessage() throws {
+        app.buttons["Show single-line tooltip below"].tap()
+        XCTAssert(app.otherElements.element(matching: NSPredicate(format: "identifier CONTAINS %@", "Tooltip with message \"This is pointing up.\"")).exists)
```

Comment:
Optional, could move some of the NSPredicates into constants, similar to ActivityIndicator? Personally found that a bit easier to read